src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -527,4 +532,162 @@ private class CRLP_RollupAccount_TEST {
         }
 
     }
+
+    @isTest
```

Comment:
I recommend moving the tests that are not actually running the full rollup jobs into a new class, maybe `CRLP_Batch_Base_TEST`. I think that test class could also use the IsParallel=true annotation.